.gitignore

```diff
@@ -25,3 +25,21 @@ build
 node_modules
 CMakeSettings.json
 xcuserdata/
+
+# Compiled executables and test files
+test/detect_backend
+test/test_libcamera
+test/test_libcamera2
+test/test_libcamera_plugin
+test/*.jpg
+test/*.png
+test/frame/
+test/test_results/
+test/libcamera_test_results/
+test/build_plugin/
```

Comment:
I propose to run experiments out of OpenCV tree and remove the test folder from git ignore list.  General compiled files looks ok.